src/main/java/org/pac4j/demo/dw/Pac4JDemoConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package org.pac4j.demo.dw;
+
+import org.pac4j.dropwizard.Pac4jFactory;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.dropwizard.Configuration;
+
+public class Pac4JDemoConfiguration extends Configuration {
```

Comment:
Each dropwizard application needs its own subclassing of Configuration, even if for simple cases they can be similar, most of the time, they will be different, see for example my configuration in an application I'm working on: [here](https://gitlab.com/victornoel/petals-cockpit/blob/master/backend/src/main/java/org/ow2/petals/cockpit/server/configuration/CockpitConfiguration.java).